src/components/TextEditor.tsx

```diff
@@ -24,6 +36,10 @@ const TextEditor: React.FC<TextEditorProps> = ({ onChange }) => {
       initialEditType="markdown"
       useCommandShortcut={true}
       theme="dark"
+      hooks={{
+        addImageBlobHook: async (blob, callback) =>
+          addImageBolb(blob, callback),
```

Comment:
Editor 컴포넌트에 hooks props.. 무엇을 뜻하는지 모르겠어요. 조금 더 자세히 변수명 정하면 좋을 것 같아요.

```suggestion
        addImageBlobHook: addImageBolb,
```